src/core/opamFilename.ml

```diff
@@ -9,6 +9,16 @@
 (*                                                                        *)
 (**************************************************************************)
 
+let is_escapable =
+  let re =
+    Re.(compile @@ seq [
+        alt [ char '/'; char '\\'];
+        str "..";
+        alt [ char '/'; char '\\']
```

Comment:
This doesn't match its description in the mli and wouldn't match on e.g. `"../../../etc/resolve.conf"`